tools/ui/src/candid.ts

```diff
@@ -77,6 +80,14 @@ export async function fetchActor(canisterId: Principal): Promise<ActorSubclass>
   }
   const dataUri = 'data:text/javascript;charset=utf-8,' + encodeURIComponent(js);
   const candid: any = await eval('import("' + dataUri + '")');
+
+  authClient = authClient ?? (await AuthClient.create())
```

Comment:
Perhaps we could deactivate the idle timeout (similar to the NNS dapp):

```suggestion
  authClient = authClient ?? (await AuthClient.create({
    idleOptions: {
      disableIdle: true,
      disableDefaultIdleCallback: true,
    },
  }))
```